src/vector/tests/mod.rs

```diff
@@ -159,7 +159,7 @@ fn test_json() {
 fn test_schema() {
     let mut ds = Dataset::open(fixture!("roads.geojson")).unwrap();
     let layer = ds.layer(0).unwrap();
-    assert_eq!(layer.name(), "OGRGeoJSON".to_string()); // change to "roads" for GDAL >= 2.2
+    assert!(layer.name() == "OGRGeoJSON" || layer.name() == "roads");
```

Comment:
Please add some comments about the two cases. I guess that someone will get confused by this :)